src/nat/llm/huggingface_llm.py

```
@@ -0,0 +1,285 @@
+    
+    def _prepare_text(self, messages):
+        """Convert messages to text using chat template or fallback."""
+        if isinstance(messages, list) and len(messages) > 0:
+            # Try using chat template
+            try:
+                text = self.tokenizer.apply_chat_template(
+                    messages,
+                    tokenize=False,
+                    add_generation_prompt=True
+                )
+            except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)